core/controllers/story_editor.py

```diff
@@ -28,19 +29,18 @@ class StoryEditorPage(base.BaseHandler):
     """The editor page for a single story."""
 
     @acl_decorators.can_edit_story
-    def get(self, topic_id, story_id):
+    def get(self, story_id, topic_id):
```

Comment:
Seems weird to reverse this. Should be topic_id then story_id?

Ditto below.

Reply:
This was needed, because I am using this same decorator in upload images as well, where the same handler is used for all entities, so in all the other decorators, the "entity_id" is the first argument (can_edit_topic, can_edit_exploration etc.). So, if I am to reuse that same controller for stories as well, the story_id was required to be the first argument.

Reply:
Another fix that can be done here is to get rid of topic_id altogether, as we have `story.corresponding_topic_id` now to get the topic_id we need.

Reply:
I'm fine with dropping topic_id altogether.

Reply:
Done